IOMC/EventVertexGenerators/python/VtxSmearedRealistic_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+# Load BetafuncEvtVtx Generator and read parameters from GT (SimBeamSpotObjectRcd)
+from IOMC.EventVertexGenerators.VtxSmearedParameters_cfi import VtxSmearedCommon
+VtxSmeared = cms.EDProducer("BetafuncEvtVtxGenerator",
+    VtxSmearedCommon
+)
+VtxSmeared.readDB = True
+
+
+
```

Comment:
```suggestion
from IOMC.EventVertexGenerators.VtxSmearedParameters_cfi import VtxSmearedCommon
VtxSmeared = cms.EDProducer("BetafuncEvtVtxGenerator",
    src = cms.InputTag("generator", "unsmeared"),
    readDB = cms.bool(True)
)
```

Reply:
Thanks Marco, this change you suggest looks cleaner indeed.
I'll add the `fillDescription` to the two generators so we can loose also the type specifications.

Reply:
solved in last push, see https://github.com/cms-sw/cmssw/pull/43242#issuecomment-1805465244